docs/cow-protocol/tutorials/widget/widget.md

```diff
@@ -187,6 +191,72 @@ Try it yourself: https://widget.cow.fi.
 
 ![Custom theme](/img/tutorials/custom-theme.png)
 
+
+## Custom tokens
+
+CowSwap uses the [token lists standard](https://tokenlists.org), which allows flexible and decentralized management of assets for trading.  
+You can manage the list of tokens in CoW Swap UI:  
+
+![Tokens management](/img/tutorials/widget-tokens-management.png)
+
+
+In the widget you have two options to add custom tokens:
+ - using `tokenLists` parameter
+ - using `customTokens` parameter
+
+### `tokenLists`
+
+In this parameter you can specify the URL or ENS address of the token list.
+
+> How to create and deploy your own token list you can find in the [guide](https://hackmd.io/@mastercow/creating-custom-token-lists).
+
+```typescript
+import { createCowSwapWidget, CowSwapWidgetParams } from '@cowprotocol/widget-lib'
+
+const container = document.getElementById('cowswap-widget')
+
+const params: CowSwapWidgetParams = {
+    tokenLists: ['https://gateway.ipfs.io/ipns/tokens.uniswap.org']
```

Comment:
Can we suggest by default a token list that's our own instead of uniswap's?

Reply:
Set https://files.cow.fi/tokens/CowSwap.json